icicle/src/hash/blake2s/extern.cu

```diff
@@ -0,0 +1,30 @@
+#include "utils/utils.h"
+#include "gpu-utils/error_handler.cuh"
+
+#include "hash/hash.cuh"
+
+#include "hash/blake2s/blake2s.cuh"
+#include "blake2s.cu"
+#include "../../merkle-tree/merkle.cu"
+#include "merkle-tree/merkle.cuh"
+
+namespace blake2s {
+  extern "C" cudaError_t blake2s_cuda(
+    BYTE* input, BYTE* output, WORD number_of_blocks, WORD input_block_size, WORD output_block_size, HashConfig& config)
+  {
```

Comment:
I noticed that the parameter order in `blake2s_cuda` is a bit different from `keccak256_cuda` (and other keccak functions). 

```cpp
keccak256_cuda(uint8_t* input, int input_block_size, int number_of_blocks, uint8_t* output, HashConfig& config)
```

To keep things consistent, so my suggestion smt like this:


```cpp
blake2s_cuda(BYTE* input, WORD input_block_size, WORD number_of_blocks, WORD output_block_size, BYTE* output, HashConfig& config)
```